src/nft/nft.cc

```diff
@@ -432,6 +432,23 @@ Nft& Nft::insert_identity(const State state, const Symbol symbol, const JumpMode
     return *this;
 }
 
+bool Nft::are_there_jump_transitions() {
```

Comment:
```suggestion
bool Nft::contains_jump_transitions() {
```
Maybe a better name? `contains` is already being used elsewhere in Mata.

Reply:
Done